tests/system/source/main.d

```diff
@@ -88,6 +114,143 @@ void main ()
         checkBlockHeight(1);
     }
 
+    /// Make Transactions for block height 2
+    {
+        API[NodeCnt] clients;
+        foreach (idx, const ref addr; Addrs)
+            clients[idx] = new RestInterfaceClient!API(addr.withSchema());
+
+        // clients[0] Create frozen transactions and enrolment
+        const Block[] blocks_1 = clients[0].getBlocksFrom(1,1);
+        assert(blocks_1[0].header.height == 1);
+
+        foreach (idx; 0 .. Block.TxsInBlock)
+        {
+            Transaction tx;
+
+            if (keypairs.length > idx)
+            {
+                Transaction freeze_tx = {
+                    type: TxType.Freeze,
+                    inputs: [Input(hashFull(blocks_1[0].txs[idx]), 0)],
+                    outputs: [Output(Amount.MinFreezeAmount,
+                        keypairs[idx].address)]
+                };
+                freeze_tx.inputs[0].signature = kp.secret.sign(hashFull(freeze_tx)[]);
+                clients[0].putTransaction(freeze_tx);
+                freeze_txs ~= freeze_tx;
+            }
+            else
+            {
+                Transaction payment_tx = {TxType.Payment, [], []};
+                payment_tx.inputs ~= Input(hashFull(blocks_1[0].txs[idx]), 0);
+                foreach (idx2; 0 .. Block.TxsInBlock)
+                    payment_tx.outputs ~= Output(Amount.MinFreezeAmount, kp.address);
+
+                payment_tx.inputs[0].signature = kp.secret.sign(hashFull(payment_tx)[]);
+                clients[1].putTransaction(payment_tx);
```

Comment:
Why are freeze transactions sent to one node and normal payment transactions sent to a different node? They will both become part of Block with height 2, right? Is there an issue when you just try to use `client[0]`? It should just gossip everything to other nodes automatically.

Reply:
It is a test. If it seems strange, the test was changed to send only to the `client[0]`